# Generated from https://pypi.org/project/datadog/
package:
  name: py3-datadog
  version: 0.49.1
  epoch: 1
  description: The Datadog Python library
  copyright:
    - license: BSD-3-Clause
  dependencies:
    runtime:
      - py3-requests
      - py3-typing

environment:
  contents:
    packages:
      - build-base
      - busybox
      - ca-certificates-bundle
      - wolfi-base
  environment:
    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
    SOURCE_DATE_EPOCH: 315532800

pipeline:
  - uses: fetch
    with:
      expected-sha256: 4cb7a7991af6cadb868fe450cd456473e65f11fc678b7d7cf61044ff1c6074d8
      uri: https://files.pythonhosted.org/packages/source/d/datadog/datadog-${{package.version}}.tar.gz

  - name: Python Build
    uses: python/build-wheel

  - uses: strip

test:
  pipeline:
    - uses: python/import
      with:
        imports: |
          import datadog
          from datadog import initialize

update:
  enabled: true
  release-monitor:
    identifier: 35391
